treeherder/webapp/api/performance_data.py

```diff
@@ -193,7 +193,7 @@ def list(self, request, project):
 
 
 class PerformanceFrameworkViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = PerformanceFramework.objects.all()
+    queryset = PerformanceFramework.objects.all().filter(enabled=True)
```

Comment:
How cool! Haven't really thought about this approach.
But I like it.

I think you can remove the `all()`, as it's redundant.

Reply:
Thanks!